mod-source-record-storage-server/src/main/java/org/folio/rest/impl/SourceStorageHandlersImpl.java

```diff
@@ -17,21 +20,24 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(SourceStorageHandlersImpl.class);
 
+  @Autowired
+  EventHandlingService instanceCreatedEventHandleService;
+
+  private String tenantId;
+
   public SourceStorageHandlersImpl(Vertx vertx, String tenantId) { //NOSONAR
+    SpringContextUtil.autowireDependencies(this, Vertx.currentContext());
+    this.tenantId = TenantTool.calculateTenantId(tenantId);
   }
 
   @Override
   public void postSourceStorageHandlersCreatedInventoryInstance(String entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     vertxContext.runOnContext(v -> {
-      try {
-        // endpoint will be implemented in https://issues.folio.org/browse/MODSOURCE-96
-        LOG.info("Event was received: {}", entity);
-        Future.succeededFuture((Response) SourceStorageHandlers.PostSourceStorageHandlersCreatedInventoryInstanceResponse.respond200())
-          .setHandler(asyncResultHandler);
-      } catch (Exception e) {
-        LOG.error("Failed to handle event", e);
-        asyncResultHandler.handle(Future.succeededFuture(ExceptionHelper.mapExceptionToResponse(e)));
-      }
+      LOG.debug("Event CREATED_INVENTORY_INSTANCE was received: {}", entity);
```

Comment:
I think it will be better to log here only entity`s id. @RuslanLavrov @OleksiiKuzminov what do you think?

Reply:
It's debug and we don't have an id

Reply:
@OleksiiKuzminov thx for your answer.